DFC.ServiceTaxonomy.ContentApproval/Controllers/ContentApprovalController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.ContentApproval.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using OrchardCore.ContentManagement;
+using OrchardCore.DisplayManagement.Notify;
+
+namespace DFC.ServiceTaxonomy.ContentApproval.Controllers
+{
+    public class ContentApprovalController : Controller
+    {
+        private readonly IAuthorizationService _authorizationService;
+        private readonly IContentManager _contentManager;
+        private readonly INotifier _notifier;
+        private readonly IHtmlLocalizer<ContentApprovalController> H;
```

Comment:
I think `IStringLocalizer` should be sufficient here.

Reply:
The notifier looks to require an IHtmlLocalizer response as an input parameter.